### ISSUE TITLE
Invalid address from sample app

### ISSUE THREAD

**Issue by stellarskye** [auth0-samples/auth0-react-native-sample#83]
I am seeing `Safari cannot open the page because the address is invalid` from the sample app - https://github.com/auth0-samples/auth0-react-native-sample/tree/master/00-Login

- Download repo and install dependencies
- Copy "Client ID" and "Domain" from Native Auth0 Application into `app/auth0-configuration.js`
- Add `com.auth0samples://YOUR_DOMAIN/ios/com.auth0samples/callback` with my domain inserted into Allowed Callback URLs
- Run ios app, enter email, clink link

<img width="397" alt="Screen Shot 2022-09-06 at 11 04 17 AM" src="https://user-images.githubusercontent.com/109759386/188707130-d91bf16a-2e1c-48d6-951a-d8f10d44e592.png">

**Comment by Widcket** [comment]
Hi @stellarskye, thanks for raising this.

Did you replace `YOUR_DOMAIN` in `com.auth0samples://YOUR_DOMAIN/ios/com.auth0samples/callback` with your Auth0 domain?

**Comment by stellarskye** [comment]
Hi @Widcket , yes I replaced `YOUR_DOMAIN` with the domain from our Auth0 application

**Comment by Widcket** [comment]
That error is because iOS cannot find an app that can be opened with the scheme of the callback URL. I'd suggest checking that the Info.plist [still contains](https://github.com/auth0-samples/auth0-react-native-sample/blob/master/00-Login/ios/Auth0Samples/Info.plist#L32) the custom scheme, and that it matches the value used in the `com.auth0samples://YOUR_DOMAIN/ios/com.auth0samples/callback` URL.

**Comment by stellarskye** [comment]
@Widcket Thank you for responding, any help is appreciated

I did not make any changes to the `Info.plist` file, the [Readme](https://github.com/auth0-samples/auth0-react-native-sample/tree/master/00-Login) does not mention this step.  What would be the change needed to this line `<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>` for the sample repo?

**Comment by Widcket** [comment]
I just tried downloading the code, and following the Readme instructions:

https://user-images.githubusercontent.com/5055789/190318092-f53589a6-cbb3-4525-ad04-170f71b207e3.mp4

**Comment by stellarskye** [comment]
@Widcket Thank you for confirming 🙏 , I will take another look at this today.  The one main difference is that our project is configured to use the passwordless email link, I'm not sure if there is any difference in the callback from the email/pw login.  Otherwise I'll double check other configuration and maybe try it against a fresh Auth0 Application/tenant.

**Comment by stellarskye** [comment]
@Widcket Yes I was able to see the same, correct behavior with a fresh tenant and application.  I then also created & configured the passwordless email connection and followed [these steps](https://auth0.com/docs/authenticate/passwordless/implement-login/universal-login) to configure the email template with the [passwordlessMethod](https://auth0.com/docs/libraries/lock/lock-configuration#passwordlessmethod-string-) set to "link".

Seems like there must be a configuration set in my original tenant/application that will need to be updated.  Will investigate on my end and post any relevant info, but for now will close this ticket.

Thanks again for the help!
